examples/Tutorial.ipynb

```diff
@@ -0,0 +1,1521 @@
+   "execution_count": 39,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loading iris/test/example_iris_features.jsonlines...           done\n",
+      "id\tprediction\n",
+      "EXAMPLE_73\tversicolor\n",
+      "EXAMPLE_18\tsetosa\n",
+      "EXAMPLE_118\tvirginica\n",
+      "EXAMPLE_78\tversicolor\n",
+      "EXAMPLE_76\tversicolor\n",
+      "EXAMPLE_31\tsetosa\n",
+      "EXAMPLE_64\tversicolor\n",
+      "EXAMPLE_141\tvirginica\n",
+      "EXAMPLE_68\tversicolor\n",
+      "EXAMPLE_82\tversicolor\n",
+      "EXAMPLE_110\tvirginica\n",
+      "EXAMPLE_12\tsetosa\n",
+      "EXAMPLE_36\tsetosa\n",
+      "EXAMPLE_9\tsetosa\n",
+      "EXAMPLE_19\tsetosa\n",
+      "EXAMPLE_56\tversicolor\n",
+      "EXAMPLE_104\tvirginica\n",
+      "EXAMPLE_69\tversicolor\n",
+      "EXAMPLE_55\tversicolor\n",
+      "EXAMPLE_132\tvirginica\n",
+      "EXAMPLE_29\tsetosa\n",
+      "EXAMPLE_127\tvirginica\n",
+      "EXAMPLE_26\tsetosa\n",
+      "EXAMPLE_128\tvirginica\n",
+      "EXAMPLE_131\tvirginica\n",
+      "EXAMPLE_145\tvirginica\n",
+      "EXAMPLE_108\tvirginica\n",
+      "EXAMPLE_143\tvirginica\n",
+      "EXAMPLE_45\tsetosa\n",
+      "EXAMPLE_30\tsetosa\n",
+      "EXAMPLE_22\tsetosa\n",
+      "EXAMPLE_15\tsetosa\n",
+      "EXAMPLE_65\tversicolor\n",
+      "EXAMPLE_11\tsetosa\n",
+      "EXAMPLE_42\tsetosa\n",
+      "EXAMPLE_146\tvirginica\n",
+      "EXAMPLE_51\tversicolor\n",
+      "EXAMPLE_27\tsetosa\n",
+      "EXAMPLE_4\tsetosa\n",
+      "EXAMPLE_32\tsetosa\n",
+      "EXAMPLE_142\tvirginica\n",
+      "EXAMPLE_85\tversicolor\n",
+      "EXAMPLE_86\tversicolor\n",
+      "EXAMPLE_16\tsetosa\n",
+      "EXAMPLE_10\tsetosa\n",
+      "EXAMPLE_81\tversicolor\n",
+      "EXAMPLE_133\tversicolor\n",
+      "EXAMPLE_137\tvirginica\n",
+      "EXAMPLE_75\tversicolor\n",
+      "EXAMPLE_109\tvirginica\n"
+     ]
+    }
+   ],
+   "source": [
+    "!generate_predictions iris/output/Iris_CV_example_iris_LogisticRegression_fold1.model iris/test/example_iris_features.jsonlines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Using SKLL via the API\n",
+    "\n",
+    "While the command line tools are intended to be the primary method of using SKLL for normal users who want to run batched machine-learning experiments, the SKLL Python API can also be useful for building applications on top of SKLL and for other advanced use cases. \n",
+    "\n",
+    "The [*learner*](https://skll.readthedocs.io/en/latest/api/learner.html) and [*data*](https://skll.readthedocs.io/en/latest/api/data.html) modules can be used to broadly replicate the functionalities of command line tools. For example, the `skll.data.Reader` class can be used to programmatically read any SKLL-compatible feature file into a SKLL [*FeatureSet*](https://skll.readthedocs.io/en/latest/api/data.html#module-skll.data.featureset) object."
```

Comment:
If we can, I think we should make the font the same for things like `skll.data.Reader` and `FeatureSet` (even if the latter is a link, which I think should still be possible). I prefer consistency here.

Reply:
So you'd prefer that it wasn't visibly a link? We won't get the underline ...

Reply:
I thought there might be something like this going on but I thought I would point it out anyway just in case it could be made uniform. I think it's fine, then.